lib/squadster_web/schema.ex

```diff
@@ -36,6 +37,20 @@ defmodule Squadster.Schema do
 
       resolve &FormationsResolver.find_squad/3
     end
+
+    @desc "Get a timetable by id"
+    field :timetables, list_of(:timetable) do
+      arg :squad_number, non_null(:string)
+
+      resolve &SchedulesResolver.find_timetables/3
+    end
+
+    @desc "Show lessons for a timetable"
+    field :lessons, list_of(:lesson) do
+      arg :timetable_id, non_null(:id)
+
+      resolve &SchedulesResolver.show_lessons/3
+    end
```

Comment:
I think we don't need this queries it'll be safer to get this data from user's squad
By the way we need to add timetables to squad in schema

Reply:
We need to discuss it

Reply:
As we discussed, we'll remove these queries